Library/Homebrew/cmd/audit.rb

```diff
@@ -1235,3 +1232,31 @@ def problem(text)
     @problems << text
   end
 end
+
+# Debugging support
```

Comment:
Okay, I refactored it to share an implementation with `Diagnostic.inject_dump_stats!`.